scripts/verify-knife

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# Verify mmfile $1 (default "set.mm") and discouraged file $2 (default
+# "discouraged") using metamath-knife.
+# Usage:
+# verify-knife [source=set.mm] [discouraged_file=discouraged]
+
+source="${1:-set.mm}"
+discouraged_file="${2:-discouraged}"
+
+metamath-knife ${source} --verify \
+                         --verify-parse-stmt \
+                         --verify-usage \
+                         --verify-markup
+
+if [ $? != 0 ]; then
+  exit 1
+fi
+
+metamath-knife ${source} -D discouraged.new >/dev/null
+diff ${discouraged_file} discouraged.new >/dev/null
+
+if [ $? != 0 ]; then
+  echo "The file '${discouraged_file}' needs regenerated."
```

Comment:
```suggestion
  echo "The file '${discouraged_file}' needs to be regenerated."
```

Reply:
Heh. This is a dialect variation of different varieties of English so I'm OK with either one.

Reply:
It's already hard enough to learn a foreign language, not to have to also learn its dialectal variations.  Please, think of us non-native speakers !

Reply:
I just went with the shorter one, but if it's less readable then I agree it's worth updating.